core/server/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -59,7 +59,7 @@
   public static final String GET_VERSION = "version";
   public static final String GET_METRICS = "metrics";
 
-  private final AlluxioWorker mWorker = AlluxioWorker.get();
+  private final AlluxioWorker mWorker = DefaultAlluxioWorker.get();
```

Comment:
I think it would make sense for the `get()` method to be part of the interface (similar to how we have `FileSystem.get()`, which returns the default implementation) as opposed the implementation